skopt/space.py

```diff
@@ -324,6 +324,13 @@ def __init__(self, dimensions):
 
         self.dimensions = _dimensions
 
+    @property
+    def is_real_space(self):
```

Comment:
`is_real` should be enough? (since typically this property will be checked through something like `space.is_real`)